### PR TITLE
fix(types): Fix error in type generation when last item is a folder

### PR DESCRIPTION
**Pull request by Palapapaa**
Sorry, in the pull request #5, I've introduced a bug that occur when the last item in the file loop is a folder.
When it's a folder I did not add a semicolon at the end of `types` var and it results in a non valid TS synthax.

### CODE REVIEW

File: src/lib/convert.ts [hunk 1]
```diff
@@ -48,6 +46,7 @@ export const convert = async (convertionOptions: ConvertionOptions): Promise<voi
         );
       }
     }
+    types = types.substring(0, types.length - 3) + ';';
```
Comment (kreuzerk) [inline src/lib/convert.ts]: Why, types.length - 3?

Comment (Palapapaa) [reply src/lib/convert.ts]: I remove the last 3 chars eg: ' | '.
Maybe it's not easy to reason about ...

Maybe we can create a var "typeDelimitor" equal to ' | ' and replace 3 by typeDelimitor.length.

Comment (kreuzerk) [reply src/lib/convert.ts]: Yes, that's a good idea - it would probably be more readable.

Comment (Palapapaa) [reply src/lib/convert.ts]: Done

**Pull request merged**